config/initializers/big_query.rb

```diff
@@ -7,8 +7,4 @@
     config.project_id  = 'teacher-vacancy-service'
     config.credentials = JSON.parse(BIG_QUERY_API_JSON_KEY)
   end
-
-  # rubocop:disable Lint/UselessAssignment
```

Comment:
Do we want to remove this now?   Per TEVA-460: "Note: We do NOT want to delete the previous Big Query work until all the new work for Big Query is completed"

Reply:
We can remove this later. It would fit the context better.

Note: We define the `big_query` variable **here** and in our **Big Query class**. 
So the definition above wasn't being used anywhere.